openedx/core/djangoapps/user_authn/tests/test_signals.py

```diff
@@ -0,0 +1,30 @@
+"""Tests for user_authn signals"""
+
+from unittest.mock import patch
+from django.test import TestCase
+
+from common.djangoapps.student.tests.factories import UserFactory, UserProfileFactory
+from openedx.core.djangolib.testing.utils import skip_unless_lms
+
+
+@skip_unless_lms
+class UserAuthnSignalTests(TestCase):
```

Comment:
These tests are new, not just moved - the `email_marketing` djangoapp didn't seem to have any tests for this signal handler.